### DISCUSSION TITLE
Such a deal-braker!

### DISCUSSION THREAD

**Discussion by PrieserMax** [simple-login/app#1988] [Kind words ❤️]
Finally I don't have to make many mail accounts anymore...
Previously I was using the cloud version, now I am switching to self-hosted -> I have to say, it is amazing!
It took me the whole night to set up, because I had some problems (which I couldn't figure out and switched to the described setup)
I first set up the official self-host image but thought about it and switched to simplelogin-ci -> So easy to switch!
A huge thanks to the devs for making this publically available and self-hostable.

**Comment by nguyenkims** [comment]
Thanks for your kind words!
